src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -47,6 +47,15 @@ static ucs_config_field_t uct_cuda_ipc_iface_config_table[] = {
      "Max number of cuda events. -1 is infinite",
      ucs_offsetof(uct_cuda_ipc_iface_config_t, max_cuda_ipc_events), UCS_CONFIG_TYPE_UINT},
 
+    {"OVERRIDE_BW", "n",
+     "Override bandwidth detection in cuda_ipc. User is expected to set UCX_CUDA_IPC_BW",
+     ucs_offsetof(uct_cuda_ipc_iface_config_t, override_bw),
+     UCS_CONFIG_TYPE_BOOL},
+
+    {"BW", "20000MBs",
+     "Effective memory bandwidth",
+     ucs_offsetof(uct_cuda_ipc_iface_config_t, bandwidth), UCS_CONFIG_TYPE_BW},
```

Comment:
Can we simplify with a single variable i.e UCX_CUDA_IPC_BW with default 0. if the user sets it non-zero then override it.